src/test/java/uk/gov/register/service/ItemValidatorTest.java

```diff
@@ -44,28 +44,68 @@ public void setup() throws NoSuchConfigException, IOException {
         MockitoAnnotations.initMocks(this);
         RegisterName registerName = new RegisterName("register");
         RegisterName countryRegisterName = new RegisterName("country");
+        final RegisterName forCurieRegisterName = new RegisterName("forCurie");
 
         when(registerMetadata.getRegisterName()).thenReturn(registerName);
         when(countryRegisterMetadata.getRegisterName()).thenReturn(countryRegisterName);
 
         when(registerMetadata.getFields()).thenReturn(Arrays.asList("register", "text", "registry", "phase", "copyright", "fields"));
-        when(countryRegisterMetadata.getFields()).thenReturn(Arrays.asList("country", "start-date"));
+        when(countryRegisterMetadata.getFields()).thenReturn(Arrays.asList("country", "start-date", "curie-info", "curie-info2"));
 
         Field textField = new Field("text", "text", registerName, Cardinality.ONE, "text text");
         Field registerField = new Field("register", "text", registerName, Cardinality.ONE, "register text");
         Field fieldsField = new Field("fields", "string", registerName, Cardinality.MANY, "fields text");
 
         Field countryField = new Field("country", "string", countryRegisterName, Cardinality.ONE, "countryName");
-        Field startDateField = new Field("start-date\"", "datetime", null, Cardinality.ONE, "Start date text");
+        Field startDateField = new Field("start-date", "datetime", null, Cardinality.ONE, "Start date text");
+        final Field curieInfoField = new Field("curie-info", "curie", forCurieRegisterName, Cardinality.ONE, "Link to curie");
```

Comment:
Could we rename these two fields to `curieFieldWithRegisterSpecified` and `curieFieldWithoutRegisterSpecified` respectively please? (or names along those lines)